runtime/src/test/java/com/dylibso/chicory/runtime/ModuleTest.java

```diff
@@ -155,36 +137,33 @@ public void shouldWorkWithStartFunction() {
                         List.of());
         var funcs = new HostFunction[] {func};
         var module =
-                Module.build(new File("src/test/resources/compiled/start.wat.wasm"))
+                Module.builder("compiled/start.wat.wasm")
+                        .build()
                         .instantiate(new HostImports(funcs));
         var start = module.getExport("_start");
         start.apply();
-        assertTrue(printer.times() > 0);
+
+        assertThat(count).hasValueGreaterThan(0);
```

Comment:
I would like to avoid adding `assertj` as a dependency only for this use case, not against in general but I believe that this is not compelling enough just yet.
```suggestion
        assertTrue(count > 0);
```